buildmaster/ATLAS_DY_7TEV_LOMASS_EXT/metadata.yaml

```diff
@@ -0,0 +1,49 @@
+setname: "ATLAS_DY_7TEV_LOMASS_EXT"
+
+version: 1
+version_comment: "Initial implementation"
+
+arXiv:
+   url: "https://arxiv.org/abs/1404.1212"
+iNSPIRE:
+   url: "https://inspirehep.net/literature/1288706"
+hepdata:
+   url: "https://www.hepdata.net/record/ins1288706"
+   version: 1
+
+nnpdf_metadata:
+   nnpdf31_process: "EWK_MLL"
```

Comment:
The `nnpdf31_process` you can find in the `plotting` file of the "old" commondata that correspond to this dataset. This is (as you correctly filled below) the `process_type`.